crypto/ecdsa/ecdsa.go

```diff
@@ -34,7 +34,7 @@ func magicHash(msg string) chainhash.Hash {
 	return chainhash.DoubleHashH(bytes)
 }
 
-func Sign(sk *btcec.PrivateKey, msg string) ([]byte, error) {
+func Sign(sk *btcec.PrivateKey, msg string) []byte {
```

Comment:
@SebastianElvis can you verify that this change does not lead to any issues? For example, we have to do other validity checks now instead of checking the error?

Reply:
Checked btcec package and this modification is fine -- `SignCompact` shall not return error anyway so they removed the returned error